### PR TITLE
Do inplace de-/serialization

### LINKED ISSUE #36: Do in-place serialization, no need to call AFF framework

### PR DESCRIPTION
**Pull request by albertmink**
fixes #36

### CODE REVIEW

File: src/zcl_aff_writer_xslt.clas.testclasses.abap [hunk 1]
```diff
@@ -942,10 +942,11 @@ CLASS ltcl_integration_test IMPLEMENTATION.
     json_writer->if_sxml_writer~set_option( option = if_sxml_writer=>co_opt_indent value = '2' ).
     CALL TRANSFORMATION (st_name) SOURCE root = test_type RESULT XML json_writer.
 
-
-    cl_aff_content_handler_factory=>get_handler_for_plain_text( )->deserialize(
```
Comment (wurzka) [inline src/zcl_aff_writer_xslt.clas.testclasses.abap:946]: The cl_aff_content_handler_factory is also called in an other place of this class. So it should be removed there, too.

**Review by wurzka — CHANGES_REQUESTED** [review]
See my comment :)

**Review by wurzka — APPROVED** [review]
LGTM :) But as there are more calls of the aff-framework, maybe we should not say that it fixes the linked issue. Or what do you think?

**Comment by albertmink** [comment]
follow up item is #45. Ok @wurzka ?

**Pull request merged**